sentinel-core/src/main/java/com/alibaba/csp/sentinel/config/SentinelConfig.java

```diff
@@ -52,12 +52,17 @@
     public static final String TOTAL_METRIC_FILE_COUNT = "csp.sentinel.metric.file.total.count";
     public static final String COLD_FACTOR = "csp.sentinel.flow.cold.factor";
     public static final String STATISTIC_MAX_RT = "csp.sentinel.statistic.max.rt";
+    /**
+     * cluster config
+     */
+    public static final String MAX_PARAM_BYTE_SIZE = "csp.sentinel.cluster.max.param.byte.size";
```

Comment:
The `SentinelConfig` itself only reserves and initializes default constants for `sentinel-core`. For Sentinel cluster module we might need to add another config getter class (like [this in transport common module](https://github.com/alibaba/Sentinel/blob/master/sentinel-transport/sentinel-transport-common/src/main/java/com/alibaba/csp/sentinel/transport/config/TransportConfig.java), which delegates the getter methods to `SentinelConfig`).

Reply:
ok, i try it.